### PR TITLE
Fixed yield display when having more than 10 yields

### PR DESCRIPTION
**Pull request by MeuhV1**

### CODE REVIEW

File: CvGameCoreDLL/CvPlot.cpp [hunk 1]
```diff
@@ -1067,23 +1067,44 @@ void CvPlot::updateSymbols()
...
+		// merijn: 5x symbols
+		if (maxYield >= 5)
 		{
 			pSymbol = addSymbol();
 			for (int iYieldType = 0; iYieldType < NUM_YIELD_TYPES; iYieldType++)
 			{
-				int iYield = yieldAmounts[iYieldType] - (maxYieldStack * i);
-				LIMIT_RANGE(0,iYield, maxYieldStack);
-				if(yieldAmounts[iYieldType])
+				if (yieldAmounts[iYieldType])
```
Comment (dguenms) [inline CvGameCoreDLL/CvPlot.cpp]: Can we make this an explicit boolean, i.e.
```
if (yieldAmounts[iYieldType] > 0)
```

File: Assets/XML/GlobalDefines.xml [hunk 2]
```diff
@@ -187,7 +187,7 @@
 	</Define>
 	<Define>
 		<DefineName>MAX_YIELD_STACK</DefineName>
-		<iDefineIntVal>5</iDefineIntVal>
+		<iDefineIntVal>10</iDefineIntVal>
```
Comment (dguenms) [inline Assets/XML/GlobalDefines.xml:190]: What does this affect?

**Review by dguenms — CHANGES_REQUESTED** [review]

File: CvGameCoreDLL/CvPlot.cpp (hunk 1 above)
Comment (MeuhV1) [reply CvGameCoreDLL/CvPlot.cpp]: No problem.

File: Assets/XML/GlobalDefines.xml (hunk 2 above)
Comment (MeuhV1) [reply Assets/XML/GlobalDefines.xml:190]: It assigns 10 "slots" to each yield instead of the old 5. The new dds file consists of 32 slots of 64x64 images.

**Comment by MeuhV1** [comment]
Updated

File: CvGameCoreDLL/CvPlot.cpp (hunk 1 above)
Comment (MeuhV1) [reply CvGameCoreDLL/CvPlot.cpp]: Done

**Review by dguenms — APPROVED** [review]

**Pull request merged**